nested/track.json

```diff
@@ -20,17 +20,59 @@
 	}, {
 		"name": "force-merge",
 		"operation-type": "force-merge"
-	},  {
+	}, {
 		"name": "randomized-nested-queries",
 		"operation-type": "search",
 		"param-source": "nested-query-source",
 		"cache": false
-	}],
+        }, {
+                "name": "randomized-term-queries",
+                "operation-type": "search",
+                "param-source": "term-query-source",
+                "cache": false
```

Comment:
Nit: If you call this `use_request_cache` and pass this value with `self._params["use_request_cache"]` in the custom parameter source, it's better aligned with Rally core.